jina/math/helper.py

```diff
@@ -49,3 +49,29 @@ def top_k(
         values = -values
 
     return values, idx
+
+
+def top_k_from_pair(
+    x_mat: 'np.ndarray',
+    x_inds: 'np.ndarray',
```

Comment:
why is `x_inds` needed?

Reply:
because we need to know the indices on both of the arrays that go inside the function (if we use this function in batches and the batches correspond to elements from a bigger array)

Reply:
but the `x` should not be batched right?